lib/mapObjects/CGHeroInstance.cpp

```diff
@@ -329,6 +325,16 @@ void CGHeroInstance::initHero(CRandomGenerator & rand)
 		levelUpAutomatically(rand);
 	}
 
+	//load base hero bonuses, TODO: per-map loading of base hero bonuses
```

Comment:
```suggestion
	// load base hero bonuses, TODO: per-map loading of base hero bonuses
	// must be done separately from global bonuses since recruitable heroes in taverns 
	// are not attached to global bonus node but need access to some global bonuses
	// e.g. MANA_PER_KNOWLEDGE for correct preview and initial state after recruit
```

Reply:
done